src/main/java/xyz/nucleoid/fantasy/RuntimeWorld.java

```diff
@@ -61,6 +62,15 @@ public void save(@Nullable ProgressListener progressListener, boolean flush, boo
         }
     }
 
+    @Override
+    protected void tickTime() {
+        if (this.shouldTickTime) {
```

Comment:
Nit: maybe a comment to clarify that this is copied from super with the function ticking removed, and why? :slightly_smiling_face: